app/(dashboard)/appSidebar.tsx

```
@@ -48,19 +49,142 @@ const settingsItems = [
+    </SidebarMenuItem>
+  );
+}
+
 export function AppSidebar() {
   const pathname = usePathname();
   const router = useRouter();
   const previousAppUrlRef = useRef<string | null>(null);
-  const { data: openCounts } = api.mailbox.openCount.useQuery();
-  const { data: mailbox } = api.mailbox.get.useQuery();
-  const isSettingsPage = pathname.startsWith(`/settings`);
+  const { data: openCounts, isLoading: isCountsLoading } = api.mailbox.openCount.useQuery();
```

ESLint

[error] 'isCountsLoading' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)